source/includes/versioned-api-snippets/pymongo-client.py

```diff
@@ -0,0 +1,7 @@
+from pymongo.mongo_client import MongoClient
+from pymongo.server_api import ServerApi
+
+// Replace <connection string> with your MongoDB deployment's connection string.
+conn_str = "<connection string>"
+
+client = pymongo.MongoClient(conn_str, server_api=ServerApi('1'), serverSelectionTimeoutMS=5000)
```

Comment:
As in the other Versioned API snippets, would it be helpful to have another comment indicating where the API Version is specified? The comment reads:
"Set the version of the Versioned API on the client."

Reply:
Good catch, will update.